Tasks/RunLoadTest/task.json

```diff
@@ -0,0 +1,89 @@
+﻿{
+  "id": "9E9DB38A-B40B-4C13-B7F0-31031C894C22",
+  "name": "CloudLoadTest",
+  "friendlyName": "Cloud-based Load Test with VSO",
+  "description": "Runs the load test in cloud; Powered by VS Online.",
+  "category": "Test",
+  "author": "Microsoft Corporation",
+  "version": {
+    "Major": 1,
+    "Minor": 0,
+    "Patch": 1
+  },
+  "demands": [
+    "msbuild"
+  ],
+  "groups": [
+    {
+      "name": "advanced",
+      "displayName": "Advanced",
+      "isExpanded": false
+    }
+  ],
+  "inputs": [
+    {
+      "name": "TestDrop",
+      "type": "filePath",
+      "label": "Load test files folder",
+      "defaultValue": "$(Build.SourcesDirectory)\\loadtest",
```

Comment:
In RM there is no Build.SourcesDirectory, we need to remove such default values. We had done cleanup of common tasks between release and build where we removed such values. Build will have templates for these instead.